### PR TITLE
Bugfix/1863 dropdown keys

### LINKED ISSUE #1863: dropdown: respond to arrow keys
Current version of Dropdown component with settings `native={false}` does not respond to arrow up/down keys to select value. 
It responds to `tab` only.

You can test it here:
https://patterns.axa.ch/?path=/story/components-molecules-dropdown--dropdown

### PR DESCRIPTION
**Pull request by danyball**
Fixes #1863 

# Done is when (DoD):
- [x] Modifications within components are reflected by tests.
- [x] A self-review of the code has been performed.
- [x] The modified component properties have been added to the typescript typings.
- [x] Potential design changes have been reviewed by a designer.
- [x] The modifications are well documented (README.md, etc.) and showcased in the stories.
- [x] The modifications are documented in the code, particularly in hard-to-understand areas (focus on **WHY**).
- [x] The modifications are shortly added to CHANGELOG.md with the issue number.
- [x] The modifications still work in IE.
- [x] The modifications are according to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)

### CODE REVIEW

File: src/components/20-molecules/dropdown/index.js [hunk 1]
```diff
@@ -156,7 +157,6 @@ class AXADropdown extends NoShadowDOM {
     // to define controlledness
     applyDefaults(this);
     // bound event handlers (so scope and de-registration work as expected)
-    this.handleWindowKeyDown = this.handleWindowKeyDown.bind(this);
```
Comment (LucaMele) [inline src/components/20-molecules/dropdown/index.js:159]: @danyball why removing this? as far as i know this is used to close the dropdown when you  click "away"

File: src/components/20-molecules/dropdown/index.js [hunk 2]
```diff
@@ -184,11 +184,52 @@ class AXADropdown extends NoShadowDOM {
...
+  handleKeyUp(e) {
     const { key, keyCode } = e;
+    e.preventDefault();
+
     if (key === 'Escape' || key === 'Esc' || keyCode === 27) {
-      e.preventDefault();
       this.openDropdown(false);
+    } else if (keyCode === 40) {
+      // arrowkey down
+      this._focusEntry('down');
```
Comment (LucaMele) [inline src/components/20-molecules/dropdown/index.js]: please use a `const KEY_DOWN = 'down'`

**Review by LucaMele — CHANGES_REQUESTED** [review]

File: src/components/20-molecules/dropdown/index.js (hunk 1 above)
Comment (danyball) [reply src/components/20-molecules/dropdown/index.js:159]: Lit html is registering those handlers with `@keydown`. Everything is working as before: https://axa-ch.github.io/plib-feature/bugfix/1863_dropdown_keys/?path=/story/components-dropdown--story

File: src/components/20-molecules/dropdown/index.js (hunk 2 above)
Comment (danyball) [reply src/components/20-molecules/dropdown/index.js]: done

**Review by LucaMele — APPROVED** [review]
Nice!

**Review by 01231 — APPROVED** [review]
LGTM

**Pull request merged**